### PR TITLE
Zero To Hero Video Series

### PR DESCRIPTION
**Pull request by VB10**
Hello dear team My name is Veli and I was Google Developer Expert last year. I have created flutter video content for 4 years for my channel as hardwareandro, and I published many contents in Flutter Commnutiy medium. I did many projects with flutter, especially in business life. So I published the concept of the zero-to-hero video, it starts from scratch and goes to professional life. This series is about 30 hours and is totaly free. still working for the Turkey community and I tried to show you how to build enterprise flutter apps, by the way, how do you think better while flutter developing.

Thank you!
Have a good work

### CODE REVIEW

**Comment by VB10** [comment]
I've signed CLA from the website

File: src/resources/videos.md [hunk 1]
```diff
@@ -158,10 +158,12 @@ To include your course, [submit a PR][]:
 * [Flutter & Dart - the Complete Guide, 2021 edition][]
 * [Flutter Crash Course][] by Nick Manning
 * [Flutter leicht gemacht 2022 - Zero to Mastery!][] by Max Berktold
+* [Flutter Zero to Hero][] by Veli Bacik
```
Comment (sfshaza2) [inline src/resources/videos.md:161]:
```suggestion
* [Flutter leicht gemacht 2022 - Zero to Mastery!][] by Max Berktold (German)
* [Flutter Zero to Hero][] by Veli Bacik (Turkish)
```

File: src/resources/videos.md [hunk 2]
```diff
@@ -158,10 +158,12 @@ To include your course, [submit a PR][]:
...
 
 [Flutter & Dart - the Complete Guide, 2021 edition]: https://www.udemy.com/course/learn-flutter-dart-to-build-ios-android-apps/
 [The Complete 2021 Flutter Development Bootcamp Using Dart]: https://www.appbrewery.co/p/flutter-development-bootcamp-with-dart/
 [Flutter Crash Course]: https://fluttercrashcourse.com/
 [Flutter leicht gemacht 2022 - Zero to Mastery!]: https://www.udemy.com/course/dart-flutter-leicht-gemacht/
+[Flutter Zero to Hero]: https://www.youtube.com/watch?v=lpvuM9lo3HU&list=PL1k5oWAuBhgXdw1BbxVGxxWRmkGB1C11l
```
Comment (sfshaza2) [inline src/resources/videos.md:167]:
```suggestion
[Flutter Zero to Hero]: {{site.youtube.com}}/watch?v=lpvuM9lo3HU&list=PL1k5oWAuBhgXdw1BbxVGxxWRmkGB1C11l
```

**Review by sfshaza2 — APPROVED** [review]
A couple minor tweaks, otherwise, lgtm

**Comment by sfshaza2** [comment]
Thanks for this, Veli!

**Comment by VB10** [comment]
@sfshaza2 thank you so mach for review. have a good work

**Comment by VB10** [comment]
Hi @domesticmouse we met at last GDE verification part, then I was an expert last year. I'd like to say hello when I see you in list. Helloo and have a good work

**Comment by domesticmouse** [comment]
> Hi @domesticmouse we met at last GDE verification part, then I was an expert last year. I'd like to say hello when I see you in list. Helloo and have a good work

Hi @VB10! 🥳

**Review by domesticmouse — CHANGES_REQUESTED** [review]
LGTM when @sfshaza2 is happy 😊

**Review by domesticmouse — APPROVED** [review]

File: src/resources/videos.md [hunk 3]
```diff
@@ -157,11 +157,13 @@ To include your course, [submit a PR][]:
...
 * [Flutter Crash Course][] by Nick Manning
-* [Flutter leicht gemacht 2022 - Zero to Mastery!][] by Max Berktold
+* [Flutter leicht gemacht 2022 - Zero to Mastery!][] by Max Berktold (German)
+* [Flutter Zero to Hero][] by Veli Bacik (Turkish)
 
 [Flutter & Dart - the Complete Guide, 2021 edition]: https://www.udemy.com/course/learn-flutter-dart-to-build-ios-android-apps/
 [The Complete 2021 Flutter Development Bootcamp Using Dart]: https://www.appbrewery.co/p/flutter-development-bootcamp-with-dart/
 [Flutter Crash Course]: https://fluttercrashcourse.com/
 [Flutter leicht gemacht 2022 - Zero to Mastery!]: https://www.udemy.com/course/dart-flutter-leicht-gemacht/
+[Flutter Zero to Hero]: {{site.youtube.com}}/watch?v=lpvuM9lo3HU&list=PL1k5oWAuBhgXdw1BbxVGxxWRmkGB1C11l
```
Comment (domesticmouse) [inline src/resources/videos.md:167]: Given the link checker errors, I'm guessing this host variable is wrong.

**Review by domesticmouse — CHANGES_REQUESTED** [review]

File: src/resources/videos.md (hunk 3 above)
Comment (VB10) [reply src/resources/videos.md:167]: @domesticmouse yep i've checked this document and i think this link could be this : {{site.youtube-site}}. i'm going to update this.

Comment (VB10) [reply src/resources/videos.md]: fixed @domesticmouse thank u

**Review by domesticmouse — APPROVED** [review]
LGTM assuming tests pass

**Pull request merged**